README.md

```diff
@@ -69,9 +69,9 @@ USB modes:
 
 OPL will automatically create the above directory structure the first time you launch it and enable your favorite device.
 
-For HDD users, OPL will read `hdd0:__common/OPL/conf_hdd.cfg` for the config entry `hdd_partition` to use as your OPL partition.
-If not found a config file, a 128Mb `+OPL` partition will be created. You can edit the config if you wish to use/create a different partition.
-All partitions created by OPL will be 128Mb (it is not recommended to enlarge partitions as it will break LBAs, instead remove and recreate manually with uLaunchELF at a larger size if needed).
+For HDD users, OPL will read `hdd0:/__common/OPL/conf_hdd.cfg` which contains one line: `hdd_partition=+OPL` to use as your OPL partition. It is recommended to use `__common`, `+OPL`, or if you have a Crystal Chip modchip: `+Crystal`.
```

Comment:
Crystal chip mention has too little use cases. Matrix users can use BOOT partition, bbnavi users can use contents partition, toxic chip users can use HDLOADER SETTINGS partitions. This doesn't mean that we should list all of them in readme. It is enough to mention that users can use any partition, even reuse any existing one.

Reply:
Thanks, I will want to consolidate the info for modchips as well at some point in the wiki with a *Note Modchips may be have differently, check this wiki page. For now will remove from here.

Reply:
Updated as requested removing Modchip info. This will be a WIP as I will be moving stuff to the wiki and link that via the readme instead.